app/plugin/jira/plugin_test.go

```diff
@@ -0,0 +1,117 @@
+package jira
+
+import (
+	"bytes"
+	"encoding/json"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"testing"
+
+	"github.com/mattermost/platform/app/plugin/plugintest"
+	"github.com/mattermost/platform/model"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
```

Comment:
making a pitch for testify here. these three subpages make tests so much easier to write and output so much more useful.